bundle/next-all-namespaces/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -1125,6 +1125,8 @@ spec:
                         value: "1"
                       - name: ALLOW_DEVWORKSPACE_ENGINE
                         value: "true"
+                      - name: CHE_ENDPOINT_MONITOR
```

Comment:
Initially it was endpoint monitor but now the logic behind this is different. 
Let's find a better name